examples/tornjak/README.md

```diff
@@ -40,3 +40,46 @@ kubectl -n spire-server port-forward service/spire-tornjak-frontend 3000:3000
 You can now access Tornjak at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
+
+## Tornjak and Ingress
```

Comment:
Since the openshift varient is documented below, perhaps we make this bit work for vanilla clusters?
```suggestion
## Tornjak and Ingress with ingress-nginx
```